components/AppFooter.vue

```diff
@@ -11,15 +11,23 @@
     <div v-else>
       <p class="text"><abbr title="United Nations Office for the Coordination of Humanitarian Affairs">OCHA</abbr> coordinates the global emergency response to save lives and protect people in humanitarian crises. We advocate for effective and principled humanitarian action by all, for all.</p>
     </div>
+    <ul class="footer-menu">
+      <li><a :href="$t('Privacy policy URL', locale)" target="_blank" rel="noopener">{{ $t('Privacy policy', locale) }}</a></li>
+      <li><a :href="$t('Copyright notice URL', locale)" target="_blank" rel="noopener">{{ $t('Copyright notice', locale) }}</a></li>
```

Comment:
I really like what you did to localize the URLs themselves, but my initial reaction was that this was an error with words being supplied to the `href` attribute. Since we can rename the "translatables" to anything we want, I'll update this to have a non-human-readable thing, such as `$('href-privacy-url', locale)` — perhaps a nice thing to document in our `locales/` directory